src/evaluatorfrontend.cpp

```diff
@@ -89,13 +89,18 @@ void EvaluatorFrontend::Help(ErrorCode errorCode) {
 	std::string helperstring =
 			"Usage: crnsimul [OPTIONS] filename"
 			"Option types:\n"
-			"	-e to to enable eulerevaluator (Required)\n"
+			" Default evaluator is eulerEvaluator\n"
+			"	-m to to enable markov evaluator\n"
 			"	-p trace parsing\n"
 			"	-s trace scanning\n"
 			"	-P enable plotting in gnuplot\n"
 			"	-O <filename> output result to file, -r required\n"
 			"	-S <step size> set stepsize for eulerevaluator\n"
 			"	-T <threshold size> set the euler evaluator threshold\n"
+			" -t to set threshold for markovEvaluator\n"
```

Comment:
Der skal vel være den samme indrykning i hele beskeden?

Reply:
jeg tror bare koden bliver ædt af format.sh så :(

Reply:
`format.sh` does not change the inside of strings, that would be insane